web/packages/shared/utils/assertUnreachable.ts

```diff
@@ -16,7 +16,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-export function assertUnreachable(x: never): never {
+export function assertUnreachable(x: any): never {
```

Comment:
think we need to revert this, i don't think this asserter will work as it intended otherwise

basically it throws an error and a ts error if there is a case in a `switch` statement, that is not handled, i think passing in any voids this